packages/vuetify/.eslintrc.js

```diff
@@ -30,10 +30,18 @@ module.exports = {
     'jest/prefer-to-contain': 'warn',
     'jest/prefer-to-have-length': 'warn'
   },
-  overrides: [{
-    files: 'dev/Playground.vue',
-    rules: {
-      'max-len': 'off'
+  overrides: [
```

Comment:
that should be rather a separate PR, also i'm not sure if this is good idea, if there's too many `it(...)` in one `describe()` maybe it's a sign that they should be split into separate `discrabe()`s

Reply:
That's fair. I'll revert this. Any suggestions on how to split up the describes, or do you know of any other test files that I could look at to get an idea?

Reply:
VIcon.spec.ts, colorUtils.spec.ts etc